### ISSUE TITLE
Issues with Typescript examples of Emotion and styled-components

### ISSUE THREAD

**Issue by Knapptr** [ben-rogerson/twin.examples#59]
Following the guide [here](https://github.com/ben-rogerson/twin.examples/tree/master/gatsby-styled-components) , I edited my `tsconfig.json` and added  `twin.d.ts`  in my root folder of the project.

Everything seems to work ( I can use `styled` when imported from `twin.macro`., and the components will render correctly. However, I'm still receiving these errors in vscode: 

![image](https://user-images.githubusercontent.com/58379702/116931243-62b9a380-ac26-11eb-8cd4-ec0026dce054.png)

![image](https://user-images.githubusercontent.com/58379702/116931323-79f89100-ac26-11eb-8c3b-fd3c007c4b82.png)

The tsconfig was created by tsc --init, and I added the `files` property after the fact (shown above)

**Comment by Knapptr** [comment]
Nevermind! I'm an idiot

**Comment by Knapptr** [comment]
`src/*/**` error was resolved by removing it from `"files":[...];`

**Comment by Knapptr** [comment]
Alright. I believe I've figured this one out using emotion- at least. My 'newness' to tsconfigs seems to have gotten in the way, and vscode was not using my tsconfig to flag errors. 

I needed to add the `./src/**/*`  to my `include` , but because I was writing it incorrectly, I thought it was wrong. Following the docs, I figured all I needed was to have `twin.d.ts` in my config. Finally! I'm so excited to start trying this out!
